custom/ewsghana/handlers/alerts.py

```diff
@@ -293,8 +293,11 @@ def handle(self):
             text = self.msg.text
 
         if not domain.commtrack_enabled:
-            print "tutaj?"
             return False
+
+        if not verified_contact.location_id:
```

Comment:
I think that in v1 inactive locations are processed in normal way but I need to agreed this with Mike